bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -168,6 +174,8 @@ def __init__(
         enforce_dtype: bool = False,
         shape: t.Optional[t.Tuple[int, ...]] = None,
         enforce_shape: bool = False,
+        from_parquet: bool = False,
```

Comment:
just a thought, maybe instead of having `from_parquet` and `to_parquet`, lets just do `parquet: bool=True|False`? It seems to me that these two fields are similar, so instead of having duplicates field we can just have one?